cmd/celestia/rpc.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/celestiaorg/celestia-node/state"
 )
 
-const authEnvKey = "CELESTIA_NODE_AUTH_TOKEN" //nolint:gosec
```

Comment:
@distractedm1nd thinkgs that that this is causing the continued linter failures, lets add back and see if we can get a pass @derrandz

Reply:
true. Removed it due to linter complaining locally, dunno how to have the local linter also detected gosec stuff but we should do it.